understanding/understanding-techniques.html

```diff
@@ -229,150 +209,134 @@ <h3>Numbered Lists, "AND"</h3>
-                        <p>they may be based on technology that is not yet stable;</p>
+                        <p>まだ安定していない技術に基づいている。</p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>they may not be <a href="conformance#accessibility-support">accessibility supported</a> in many cases (for example, assistive technologies do not work with them yet);
+                        <p>多くの場合において<a href="conformance#accessibility-support">アクセシビリティ サポーテッド</a>でない (例えば、支援技術がテクニックに対応できていない)。
                         </p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>they may not be testable;</p>
+                        <p>テスト可能でない。</p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>in some circumstances they may not be applicable or practical, and may even decrease
-                           accessibility for some users while increasing it for others;
+                        <p>ある状況においてそのテクニックが適用可能でないあるいは実用的でなく、かつその他の利用者に対してはアクセシビリティを向上させる一方、一部の利用者に対してアクセシビリティを低下させる可能性がある。
                         </p>
                         				
                      </li>
                      				
                      <li>
                         					
-                        <p>they may not address the success criterion itself, and instead provide related accessibility
-                           benefits.
+                        <p>達成基準そのものには対処しておらず、代わりに関連するアクセシビリティ上の利点を提供している。
                         </p>
                         				
                      </li>
                      			
                   </ul>
                   			
-                  <p>Authors are encouraged to apply all of the techniques where appropriate to best address
-                     the widest range of users' needs.
+                  <p>コンテンツ制作者には、最も幅広い利用者のニーズに対処するのに最適なすべてのテクニックを適用することが推奨される。
                   </p>
                   		
                </section>
                		
                <section id="failures">
                   			
-                  <h2>Failures</h2>
+                  <h2>失敗例</h2>
                   			
-                  <p><em>Failures</em> are things that cause accessibility barriers and fail specific success criteria.
-                     The documented <em>failures</em> are useful for:
+                  <p><em>失敗例</em>はアクセシビリティ上のバリアを引き起こし、特定の達成基準への不適合を招くものである。文書化された<em>失敗例</em>は以下の点で有益である:
                   </p>
                   			
                   <ul>
                      				
                      <li>
                         					
-                        <p>Authors to know what to avoid,</p>
+                        <p>コンテンツ制作者にとっては、避けるべきことを知ることができる。</p>
```

Comment:
原文がカンマ終わりなのを尊重するならこうでしょうか。

```suggestion
                        <p>コンテンツ制作者にとっては、避けるべきことを知ることができ、</p>
```

Reply:
取り込みました。